aiida/orm/data/structure.py

```diff
@@ -1160,17 +1148,34 @@ def append_site(self, site):
                 "The StructureData object cannot be modified, "
                 "it has already been stored")
 
-        new_site = Site(site=site)  # So we make a copy
+        #new_site = Site(site=site)  # So we make a copy
```

Comment:
@nmounet You're right, I'll see if I can pull this file out of the commit somehow and push it separately.